packages/foreman/rubygem-foreman_maintain/rubygem-foreman_maintain.spec

```diff
@@ -25,6 +25,8 @@ BuildRequires: %{?scl_prefix_ruby}ruby(rubygems)
 BuildRequires: %{?scl_prefix_ruby}ruby
 BuildArch: noarch
 Provides: %{?scl_prefix}rubygem(%{gem_name}) = %{version}
+Provides: %{pkg_name} = %{version}
```

Comment:
I don't know everything about RPM packaging but this feels redundant. Shouldn't it actually be ```%{gem_name}``` if you want to make ```foreman_maintain``` work? From the koji build log:
```
Provides: config(rubygem-foreman_maintain) = 0.1.3-1.el7 foreman-maintain = 0.1.3 rubygem(foreman_maintain) = 0.1.3 rubygem-foreman_maintain = 0.1.3 rubygem-foreman_maintain = 0.1.3-1.el7
```

Reply:
ah, yes, thanks @ekohl :+1: updated

Reply:
That's what I would expect:

```
Provides: config(rubygem-foreman_maintain) = 0.1.3-1.el7 foreman-maintain = 0.1.3 foreman_maintain = 0.1.3 rubygem(foreman_maintain) = 0.1.3 rubygem-foreman_maintain = 0.1.3-1.el7
```